### PR TITLE
[Intl] Made countries ISO 3166 compliant + exclude Zzzz script code

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no     
| Deprecations? | no
| Tests pass?   | yes (including intl-data group)
| Fixed tickets | https://github.com/symfony/symfony/pull/31350#issuecomment-488771154, #18613
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

For consistency :)

### CODE REVIEW

File: src/Symfony/Component/Intl/Data/Generator/RegionDataGenerator.php [hunk 1]
```diff
@@ -27,32 +27,20 @@
...
-     */
     private static $blacklist = [
-        self::UNKNOWN_REGION_ID => true,
         // Look like countries, but are sub-continents
-        self::OUTLYING_OCEANIA_REGION_ID => true,
-        self::EUROPEAN_UNION_ID => true,
-        self::EUROZONE_ID => true,
-        self::UNITED_NATIONS_ID => true,
-        // No longer exists
-        self::NETHERLANDS_ANTILLES_ID => true,
```
Comment (ro0NL) [inline src/Symfony/Component/Intl/Data/Generator/RegionDataGenerator.php]: removed upstream :+1:

**Comment by ro0NL** [comment]
See https://github.com/symfony/symfony/pull/31350#issuecomment-489572060

it seems there are a few more codes that should be excluded before we comply with ISO-3166

**Comment by fabpot** [comment]
@ro0NL Can you rebase this one please?

**Comment by ro0NL** [comment]
@fabpot  done.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @ro0NL.

**Pull request merged**